.github/workflows/check.yml

```
@@ -27,7 +27,8 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - name: ts-check
         run: |
-          yarn install --frozen-lockfile
+          yarn install
```

Can we keep the frozen lockfile setting?

```suggestion
          yarn install --frozen-lockfile
```

---

added it back